File: Data Structures & Algorithms/concatenation-of-array/submission-6.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        ArrayList<Integer> res = new ArrayList<>();
        for (int i = 0; i < 2; i++) {
            for (int j = 0; j < nums.length; j++) {
                res.add(nums[j]);
            }
        }
        int[] res2 = new int[res.size()];
        for (int i = 0; i < res.size(); i++) {
            res2[i] = res.get(i);
        }
        return res2;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-6.py
from collections import Counter 
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        counter = Counter(nums)
        answer = [] 
        res = [[] for i in range(len(nums) + 1)]

        for num, freq in counter.items():
            res[freq].append(num) 

        for i in range(len(res) - 1, 0, -1):
            for j in res[i]:
                answer.append(j) 
                if len(answer) == k:
                    return answer 
